stage1/init/pod_test.go

```diff
@@ -50,8 +50,11 @@ func TestQuoteExec(t *testing.T) {
 			input:  []string{`path with "quo't'es" and \slashes`, `"arg"`, `\`},
 			output: `"path with \"quo\'t\'es\" and \\slashes" "\"arg\"" "\\"`,
 		}, {
-			input:  []string{`$path$`},
-			output: `"$path$"`,
+			input:  []string{`$path$`, `$argument`},
+			output: `"$path$" "$$argument"`,
+		}, {
+			input:  []string{`path`, `Args\nwith\nnewlines`},
```

Comment:
I thought I commented on this but it disappeared?

It should be
```
input:  []string{`path`, "Args\nwith\nnewlines"},
```
because the test is about testing new lines rather than testing the two characters `\` and `n`. If we were testing from a json file, it would write `\n` because that's how new lines are marshaled in json files.

I am not sure what should be the output expected.